app.js

```diff
@@ -0,0 +1,74 @@
+const addMovieBtn = document.getElementById("add-movie-btn");
+const searchBtn = document.getElementById("search-btn");
+
+const movies = [];
+
+const filtermovieHandler = () => {
+  const filteredElement = document.getElementById("filter-title").value;
+  renderMovieElement(filteredElement);
+};
+
+const renderMovieElement = (filter = "") => {
+  const movieList = document.getElementById("movie-list");
+
+  if (movies.length === 0) {
+    movieList.classList.remove("visible");
+    return;
+  } else {
+    movieList.classList.add("visible");
+  }
+
+  movieList.innerHTML = "";
+
+  const filtermovieElement = !filter
+    ? movies
+    : movies.filter((movie) => movie.info.title.includes(filter));
+
+  filtermovieElement.forEach((movie) => {
+    const movieLi = document.createElement("li");
```

Comment:
movieList ?

Reply:
i've already used `movieList` for an `id` of `movie-list`. Hence named here like`movieLi` !! if you `movieElement` would be more appropriate then i will change it to `movieElement` ?? @faran4engg

Reply:
> i've already used `movieList` for an `id` of `movie-list`. Hence named here like`movieLi` !! if you `movieElement` would be more appropriate then i will change it to `movieElement` ?? @faran4engg

But movieLi, this name is not making sense. Also it is blocked scope so won't be a problem.

Reply:
changed name to `movieListItem` @faran4engg